app/assets/stylesheets/assignments/calendar.scss

```diff
@@ -0,0 +1,73 @@
+#calender {
+  border-collapse: collapse;
+  border-spacing: 0;
+  width: 100%;
+  height: 100%;
+  min-width: 800px;
+  margin-bottom: 5%;
+}
+
+.cal-head {
+  text-align: right;
+  color: #999;
+  font-weight: normal;
+}
+
+.cal-cell {
+  width: 14%;
+}
+
+.date {
+  text-align: right;
+  display: block;
+  height: 0;
+  font-size: 0.8em;
+  padding: 0.25em 0.75em 73% 0.25em;
+  position: relative;
+  border-top: 1px solid #DEDBDB;
```

Comment:
I originally thought it should be capitalized similar to SQL but have since realized it's rarely capitalized in practice.